persistence/javadsl/src/main/scala/com/lightbend/lagom/internal/javadsl/persistence/AbstractPersistentEntityRegistry.scala

```diff
@@ -133,7 +134,7 @@ abstract class AbstractPersistentEntityRegistry(system: ActorSystem, injector: I
           .asJava
 
       case None =>
-        throw new UnsupportedOperationException(s"The $journalId Lagom persistence plugin does not support streaming events by tag")
+        throw new UnsupportedOperationException(s"The Lagom persistence plugin does not support streaming events by tag")
```

Comment:
@renatocaval's suggestion sounds like an improvement to me.

Currently, there is a stubbed implementation in the the tests (`AbstractPersistentEntityRegistrySpec` for Java and Scala) that does not define `queryPluginId` or `eventsByTagQuery`. I think it could be replaced by a stub implementation. The tests don't actually cover this method anyway.

I think this is an unnecessary coupling of unrelated functionality (registration/lookup of entities and providing an event stream) but splitting them up would be a bigger change we would have to address separately.

In the meantime, @patrick-premont if you'd like to make the refactoring @renatocaval suggested in this pull request, go right ahead, but IMHO it's optional (no pun intended).